docs/hoon/library/2h.md

```diff
@@ -152,141 +237,189 @@ Source
         ?:  (vor n.l.a n.r.a)
           [n.l.a l.l.a $(l.a r.l.a)]
         [n.r.a $(r.a l.r.a) r.r.a]
-      ::
 
 
-Examples
---------
 
-    > =b (sa `(list @t)`['a' 'b' 'c' ~])
+#### Examples
+
+    > =b (sy `(list @t)`['a' 'b' 'c' ~])
     > (~(del in b) 'a')
     {'c' 'b'}
-    > =b (sa `(list @t)`['john' 'bonita' 'daniel' 'madeleine' ~])
+
+    > =b (sy `(list @t)`['john' 'bonita' 'daniel' 'madeleine' ~])
     > (~(del in b) 'john')
     {'bonita' 'madeleine' 'daniel'}
+
     > (~(del in b) 'susan')
     {'bonita' 'madeleine' 'daniel' 'john'}
 
-------------------------------------------------------------------------
 
+***
+###`+-dif:in`
+
```

Comment:
Get a space between `###` and `+-dif:in`.  It doesn't render correctly without it.